src/javascript/app_2/components/elements/menu_drawer.jsx

```diff
@@ -5,36 +5,43 @@ import { DrawerItems, DrawerItem } from '../elements/drawer/index.jsx';
 import { requestLogout } from '../../base/common';
 import Client from '../../../_common/base/client_base';
 
-export const MenuDrawer = () => (
-    <div className='drawer-items-container'>
-        <PerfectScrollbar>
-            <div className='list-items-container'>
-                <DrawerItem text={localize('Manage Password')} />
-                <DrawerItem text={localize('Useful Resources')}/>
-                <DrawerItem text={localize('Login History')}/>
-                <hr />
-                <DrawerItems
-                    text={localize('Settings')}
-                    items={[
-                        { text: localize('Personal Detail') },
-                        { text: localize('Account Authentication') },
-                        { text: localize('Financial Assessment') },
-                        { text: localize('Professional Trader') },
-                        { text: localize('Self Exclusion') },
-                        { text: localize('Trading Limits') },
-                        { text: localize('Authorised Applications') },
-                        { text: localize('API Token') },
-                    ]}
-                />
-                <DrawerItem text={localize('Purchase Confirmation')} />
-                <DrawerItem text={localize('Purchase Lock')} />
-                <DrawerItem text={localize('Dark Theme')} />
-                <hr />
-                <DrawerItem text={localize('Contact Us')}/>
-                {Client.isLoggedIn() &&
-                    <DrawerItem text={localize('Logout')} custom_action={requestLogout}/>
-                }
-            </div>
-        </PerfectScrollbar>
-    </div>
-);
+export const MenuDrawer = () => {
+    const is_mobile = window.innerWidth < 980;
```

Comment:
isn't that more of a tablet size, than a mobile size?

Reply:
What if I changed it to `is_desktop = window.innerWidth > 979` instead and used `!is_desktop && ..` on L31?
We don't really have a concrete reference for tablet sizes in `trading_app`.

Reply:
sounds fine to me